Corona-Warn-App/src/main/res/layout/family_pcr_test_card_positiv.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/title"
+        style="@style/headline6"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/card_padding"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="@dimen/spacing_small"
+        android:focusable="false"
```

Comment:
is that required ? I see it in all texts in all items layouts

Reply:
To be honest: I'm not sure - just used the same approach we used for submission cards, as it successfully passed accessibility tests 👀